packages/react/src/themes/teams/index.tsx

```diff
@@ -37,16 +37,16 @@ const themeIcons: ThemeIcons = Object.keys(
 
 const icons: ThemeIcons = {
   ...themeIcons,
-  'stardust-checkmark': themeIcons['accept'],
-  'stardust-circle': themeIcons['stardust-circle'],
-  'stardust-close': themeIcons['close'],
-  'stardust-arrow-up': themeIcons['triangle-up'],
-  'stardust-arrow-down': themeIcons['triangle-down'],
-  'stardust-arrow-end': themeIcons['triangle-right'],
-  'stardust-menu-arrow-down': themeIcons['chevron-down-medium'],
-  'stardust-menu-arrow-end': themeIcons['chevron-right-medium'],
-  'stardust-pause': themeIcons['pause'],
-  'stardust-play': themeIcons['play'],
+  'icon-checkmark': themeIcons['accept'],
+  'icon-circle': themeIcons['icon-circle'],
```

Comment:
Not sure I understand this line

Reply:
Looks like this is a teams theme icon literally named `iconCircle`.